src/SystemStatus/SiteData.php

```diff
@@ -43,6 +43,6 @@ function ($version, $project) use ($coreVersion) {
             ARRAY_FILTER_USE_BOTH
         );
 
-        return array_filter($filteredVersionMap);
+        return $filteredVersionMap;
```

Comment:
With this change the function signature with `@return string[]` no longer holds since entries can be null. How about updating it to [`@return array<string, string|null>`](https://phpstan.org/writing-php-code/phpdoc-types)?

It might also warrant a proper DocBlock comment even though that is not very widely used in the current state of the project.

Reply:
Fixed.